### ISSUE TITLE
Bug: `kubeapilinter` incorrectly flags field comment with leading space

### ISSUE THREAD

**Issue by camilamacedo86** [kubernetes-sigs/kube-api-linter#107]
The linter reports:

```
commentstart: godoc for field Foo should start with 'foo ...'
```

for:

```go
// Foo is an example field of Admiral.
Foo string `json:"foo,omitempty"`
```

This is a false positive.

### Expected

The comment starting with `// Foo ...` is correct and should not be flagged.

### Context

This format is standard and expected by:

- `gofmt`, `go vet`
- Effective Go
- `golangci-lint`, `revive`, `staticcheck`

Omitting the space (i.e. `//Foo`) breaks formatting and style guides.

### Ask

Update the `commentstart` rule to allow standard `// Foo` format.

Thanks!

**Comment by everettraven** [comment]
This is not a bug based on the details of the `commentstart` rule: https://github.com/kubernetes-sigs/kube-api-linter/blob/e719da12d8403e66335882c7ad704f8e95927ce0/pkg/analysis/commentstart/analyzer.go#L35-L42

It explicitly states in the rule that it checks that comments start with the serialized name of the field. The error you are seeing is not related to the leading space, but instead the casing.

It is asking for:
```go
// foo is an example field of Admiral. <--- Serialized name in use!
Foo string `json:"foo,omitempty"`
```
